cmd2/cmd2.py

```diff
@@ -3026,7 +3036,6 @@ def do_py(self, args: argparse.Namespace) -> bool:
         if self._in_py:
             err = "Recursively entering interactive Python consoles is not allowed."
             self.perror(err, traceback_war=False)
-            self._last_result = CommandResult('', err)
```

Comment:
CommandResult is now an unused import which needs to be removed so flake8 checks will pass